docs/staking.md

```diff
@@ -0,0 +1,122 @@
+---
+title: Staking and Adaptive Issuance
+author: Alireza Haghshenas
+---
+
+# Staking and Adaptive Issuance
+
+Staking is an update to the "Proof of Stake" Mechanism in Tezos.
+
+Before Staking, the only way for Tez holders to participate in the network was by delegating their tokens to a baker. Delegated funds contributed to a delegate's voting and baking power, but were not subject to staking: they could not be slashed in case the baker misbehaved. This is a useful feature for many users, but limits the total amount ot staked tez to the amount of tez that bakers themselves hold. In other words, if participating in the "Proof of Stake" mechanism required setting up and maintaining a baker, which is a significant barrier to entry for many users.
+
+Staking changes this by allowing any user to participate in the "Proof of Stake" mechanism without setting up a baker. Users can now stake their tokens directly, and their staked tokens will be subject to slashing in case of misbehavior. This allows the total amount of staked tez to be much higher than the amount of tez that bakers themselves hold, which in turn increases the security of the network.
+
+Users can control their staked funds using the `stake`, `unstake`, and `finalize_unstake` operations. These are implemented as pseudo-entrypoints, and the parameters are passed to a transfer operation with the same destination as the sender.
+
+To accept staked funds, a delegate needs to have opted in to staking. This is done by `set_delegate_parameters`. This part is not supported by Taquito, as it is not a dApp operation. More information [here](https://tezos.gitlab.io/paris/adaptive_issuance.html#staking-policy-configuration).
+
+# Staking Funds
+To stake your funds, you need to call the `stake` operation.
+You need yo have delegated your funds to a baker before you can stake them.
```

Comment:
> You need yo have delegated your funds to a baker before you can stake them.
might be clearer to say you need to set delegate to delegate your balance to them before you can stake with them? 
also might worth mentioning checking their delegate accept staking?